packages/sdk/src/addresses/index.ts

```diff
@@ -143,18 +160,19 @@ export function getAccountDataFromHdNode({
   return accountData
 }
 
-export function getAllAccountsFromHdNode({ hdNode, network = "testnet" }: GetAllAccountsFromHDNodeOptions) {
+export function getAllAccountsFromHdNode({ hdNode, network = "testnet", account = 0, addressIndex = 0 }: GetAllAccountsFromHDNodeOptions) {
   const accounts: Account[] = []
   const addressTypesList = Object.values(addressTypeToName) as AddressFormats[]
 
   addressTypesList.forEach((addrType) => {
-    const account = getAccountDataFromHdNode({
+    const walletAccount = getAccountDataFromHdNode({
       hdNode,
       format: addrType,
-      network
+      network,
+      account, addressIndex
```

Comment:
lets run lint here

Reply:
I don't think there's any linting/formatting here hahaha but I just added spaces

Reply:
i mean`addressIndex` is supposed to be on the next line right? i think the formatting is not right 🤔